sandbox/src/mocks/r4/createServiceRequest/requests/createIncomplete.json

```diff
@@ -0,0 +1,87 @@
+{
+  "resourceType": "ServiceRequest",
+  "subject": {
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9912003888"
+    }
+  },
+  "category": [
+    {
+      "coding": [
+        {
+          "system": "https://fhir.nhs.uk/CodeSystem/message-category-servicerequest",
+          "code": "referral"
+        }
+      ]
+    }
+  ],
+  "code": {
+    "coding": [
+      {
+        "system": "https://fhir.nhs.uk/CodeSystem/ServiceRequest-Code",
+        "code": "advice-and-guidance"
+      }
+    ]
+  },
+  "priority": "routine",
+  "intent": "order",
```

Comment:
I see there is a question [Question](https://nhsd-confluence.digital.nhs.uk/pages/viewpage.action?pageId=534350452) for this already. I agree we should check again. Although "order" was picked when we discussed in the context of Patient Care APIs